src/test/java/org/openmrs/contrib/qaframework/automation/ManageUserAccountSteps.java

```diff
@@ -0,0 +1,136 @@
+		userAccountPage.enterUsername("dclerk");
+		userAccountPage.setUserPrivilegeLevel("High");
+		userAccountPage.setUserPassword("Clerk123", "Clerk123");
+		assertFalse(userAccountPage.containsText(SYSTEM_ALERT));
+		List<String> validationErrors = userAccountPage.getValidationErrors();
+		assertTrue(userAccountPage.isConfirmPasswordMatching(validationErrors));
+		userAccountPage.selectConfiguresForms();
+		userAccountPage.selectRecordsAllergies();
+		userAccountPage.selectRegistersPatients();
+		userAccountPage.selectUsesPatientSummary();
+		userAccountPage.addProviderDetails();
+		userAccountPage.setUserIdentifier("C010");
+		userAccountPage.setUserProviderRole("Clerk");
+	}
+
+	@When("a user fills details of the Nurse in the user account form")
+	public void registerNurseDetails() {
+		userAccountPage.enterPersonalDetails("Super", "Nurse");
+		userAccountPage.selectGender();
+		userAccountPage.clickOnAddUserAccount();
+		userAccountPage.enterUsername("snurse");
+		userAccountPage.setUserPrivilegeLevel("High");
+		userAccountPage.setUserPassword("Nurse123", "Nurse123");
+		assertFalse(userAccountPage.containsText(SYSTEM_ALERT));
+		List<String> validationErrors = userAccountPage.getValidationErrors();
+		assertTrue(userAccountPage.isConfirmPasswordMatching(validationErrors));
+		userAccountPage.selectAdministersSystem();
+		userAccountPage.selectEntersVitals();
+		userAccountPage.selectRegistersPatients();
+		userAccountPage.selectUsesPatientSummary();
+		userAccountPage.addProviderDetails();
+		userAccountPage.setUserIdentifier("N101");
+		userAccountPage.setUserProviderRole("Nurse");
+	}
+
+	@When("a user fills details of the Doctor in the user account form")
+	public void registerDoctorDetails() {
+		userAccountPage.enterPersonalDetails("Super", "Doctor");
+		userAccountPage.selectGender();
+		userAccountPage.clickOnAddUserAccount();
+		userAccountPage.enterUsername("sdoctor");
+		userAccountPage.setUserPrivilegeLevel("Full");
+		userAccountPage.setUserPassword("Doctor123", "Doctor123");
+		assertFalse(userAccountPage.containsText(SYSTEM_ALERT));
+		List<String> validationErrors = userAccountPage.getValidationErrors();
+		assertTrue(userAccountPage.isConfirmPasswordMatching(validationErrors));
+		userAccountPage.selectAdministersSystem();
+		userAccountPage.selectConfiguresForms();
+		userAccountPage.selectConfiguresMetadata();
+		userAccountPage.selectSchedulesAndOverbooksAppointments();
+		userAccountPage.selectHasSuperUserPrivileges();
+		userAccountPage.addProviderDetails();
+		userAccountPage.setUserIdentifier("D050");
+		userAccountPage.setUserProviderRole("Clinical Doctor");
+	}
+
+	@And("a user clicks on save account button")
+	public void clickOnSaveUserAccount() {
+		userAccountPage.saveUserAccount();
+		userAccountPage.waitForPage();
+	}
+
+	@Then("the system adds user account into the users table")
+	public void systemAddsUserAccount() {
+		assertTrue(textExists("Account Saved Successfully"));
+	}
+}
```

Comment:
Well done @kdaud, Adding another empty line would make git very happy in future for code maintainability

Reply:
> Adding another empty line would make git very happy in future for code maintainability

AFAIK we only need one empty line at the end of the file to cater for collaborators! This silent line was catered for at line code [137](https://github.com/kdaud/openmrs-contrib-qaframework/edit/RATEST-157/src/test/java/org/openmrs/contrib/qaframework/automation/ManageUserAccountSteps.java?pr=%2Fopenmrs%2Fopenmrs-contrib-qaframework%2Fpull%2F59)